src/components/DateTimePicker/DateTimePicker.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Picker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+const DateTimePicker = props => {
+  const { date, setDate, customInput, customInputProps, isTimeOnly, isSmallScreen, ...rest } = props;
+
+  const fixOnEnter = e => {
+    // makes sure input loses focus on enter, to avoid date parsing bugs
+    if (e.key.toLowerCase() === "enter") {
+      setTimeout(() => document.activeElement.blur());
+    }
+  };
+
+  const handleTimeChange = newTime => {
+    if (!newTime) return;
```

Comment:
newTime can either be an instance of Date or null (in case we clear the input). this makes sure we dont call date methods on null (getHours, getMinutes). although i agree we need some fallback time for that case, perhaps 00:00 ?